src/devices/tuya.ts

```diff
@@ -6359,7 +6359,7 @@ const definitions: Definition[] = [
         toZigbee: [],
         configure: tuya.configureMagicPacket,
         exposes: [e.battery(), e.temperature(), e.humidity(), e.illuminance()],
-        whiteLabel: [tuya.whitelabel('Tuya', 'QT-07S', 'Soil sensor', ['_TZ3000_kky16aay', '_TZE204_myd45weu'])],
+        whiteLabel: [tuya.whitelabel('Tuya', 'TS0222_temperature_humidity', 'Soil sensor', ['_TZ3000_kky16aay', '_TZE204_myd45weu'])],
```

Comment:
```suggestion
        whiteLabel: [tuya.whitelabel('Tuya', 'QT-07S', 'Soil sensor', ['_TZE204_myd45weu'])],
```

This would make `_TZ3000_kky16aay` be detected as `TS0222_temperature_humidity`